specifyweb/stored_queries/tests.py

```diff
@@ -750,6 +750,7 @@ def test_date_part_filter_combined(self):
     #     self.assertEqual(params, (7, 1, 2, 8, 1, 2))
 
 
+@skip("This test doesn't run in GitHub Actions.")
```

Comment:
Don't skip this test. It checks the sqlalchemy model vs django model (and not specify6 vs 7). This is bc in the past, we've had issues where sqlalchemy model was different than django, leading to incorrect queries. It is an internal test more than anything

Reply:
Sorry, didn't mean to push that.  That was just a temporary local change.  Will revert.